src/models.rs

```diff
@@ -26,28 +26,33 @@ pub(crate) struct Configuration {
 
 #[derive(Debug, Deserialize)]
 pub(crate) struct Environment {
-    name: String,
+    #[serde(rename = "name")]
+    _name: String,
     pub environment_id: String,
     pub features: Vec<Feature>,
     pub properties: Vec<Property>,
 }
 
 #[derive(Clone, Debug, Deserialize)]
 pub(crate) struct Segment {
-    pub name: String,
+    #[serde(rename = "name")]
+    pub _name: String,
     pub segment_id: String,
-    pub description: String,
-    pub tags: Option<String>,
+    #[serde(rename = "description")]
+    pub _description: String,
+    #[serde(rename = "tags")]
+    pub _tags: Option<String>,
     pub rules: Vec<SegmentRule>,
 }
 
 #[derive(Debug, Clone, Deserialize)]
 pub(crate) struct Feature {
     pub name: String,
     pub feature_id: String,
-    #[serde(rename(deserialize = "type"))]
```

Comment:
Let's use the same name to serialize and deserialize. This way, if we want to, we can execute a roundtrip test